integration-test/list_test.go

```diff
@@ -36,3 +51,49 @@ func (s *RunVSuite) TestListSleep(c *check.C) {
 	c.Assert(out, checker.Contains, ctrName)
 	<-exitChan
 }
+
+func (s *RunVSuite) TestListSleepJson(c *check.C) {
+	//TODO: enable this after fixing!!!
+	//c.Skip("enable this after fixing")
```

Comment:
Hmmm... I don't know as well.
I just find every test case has it at the beginning, so I write it there.